golangci.yaml

```
@@ -0,0 +1,44 @@
+---
+run:
+  go: '1.21'
```

```suggestion
  go: '1.22'
```

Just so that it is consistent with the `go.mod` file 🙂